deepfence_frontend/apps/dashboard/src/features/postures/pages/Accounts.tsx

```diff
@@ -710,55 +716,20 @@ const Accounts = () => {
                   <FilterHeader onReset={onResetFilters} />
                   <div className="flex flex-col gap-y-6 p-4">
                     <fieldset>
-                      <legend className="text-sm font-medium">Active</legend>
                       <div className="flex gap-x-4 mt-1">
                         <Checkbox
-                          label="No"
-                          checked={searchParams.getAll('active').includes('true')}
+                          label="Active"
+                          checked={searchParams.get('active') === 'true'}
                           onCheckedChange={(state) => {
                             if (state) {
                               setSearchParams((prev) => {
-                                prev.append('active', 'true');
-                                prev.delete('page');
-                                return prev;
-                              });
-                            } else {
-                              setSearchParams((prev) => {
-                                const prevStatuses = prev.getAll('active');
-                                prev.delete('active');
-                                prevStatuses
-                                  .filter((active) => active !== 'true')
-                                  .forEach((active) => {
-                                    prev.append('active', active);
-                                  });
                                 prev.delete('active');
-                                prev.delete('page');
-                                return prev;
-                              });
-                            }
-                          }}
-                        />
-                        <Checkbox
-                          label="Yes"
-                          checked={searchParams.getAll('active').includes('false')}
-                          onCheckedChange={(state) => {
-                            if (state) {
-                              setSearchParams((prev) => {
-                                prev.append('active', 'false');
-                                prev.delete('page');
+                                prev.append('active', 'true');
```

Comment:
we need to reset pagination state on filters, so prev.delete('page'); is actually required.